src/model_ops_manager/mlflow_agent/configuration.py

```diff
@@ -3,14 +3,18 @@
 
 class MLFlowConfiguration:
 
-    @staticmethod
-    def set_tracking_uri(tracking_uri):
+    _mlflow_connect_tracking_server_uri = None
```

Comment:
To make the tracking server URI a class-level field that can be checked if needed.